app/layout.tsx

```diff
@@ -6,10 +6,10 @@ export const metadata: Metadata = {
   description: 'Generated by create next app',
 };
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
+export default function RootLayout({ children }: { children: React.RactNode }) {
   return (
     <html lang='en'>
-      <body>{children}</body>
+      <body>{chidren}</body>
```

Comment:
```suggestion
      <body>{children}</body>
```
Children was misspelled.